call.go

```diff
@@ -39,19 +39,20 @@ func (cc *ClientConn) Invoke(ctx context.Context, method string, args, reply int
 		interceptors = append([]UnaryClientInterceptor{cc.dopts.unaryInt}, interceptors...)
 	}
 
-	return getChainInvoker(interceptors, invoke)(ctx, method, args, reply, cc, opts...)
+	if len(interceptors) > 0 {
+		return interceptors[0](ctx, method, args, reply, cc, getChainInvoker(interceptors, 0, invoke), opts...)
```

Comment:
This still re-chains the interceptors for each RPC.
Chain can done in Dial (either while specifying dial options, or after processing all dial options). Then there will be only one interceptor, and can be used by all RPCs.

Reply:
I see... I misunderstood it before. I have moved the chaining logic to clientconn init time.